include/kframework/evm-types.md

```diff
@@ -281,7 +280,7 @@ A cons-list is used for the EVM wordstack.
 
 ```k
     syntax Int ::= #sizeWordStack ( WordStack )       [function, total, smtlib(sizeWordStack)]
-                 | #sizeWordStack ( WordStack , Int ) [function, total, klabel(sizeWordStackAux), smtlib(sizeWordStackAux)]
+                 | #sizeWordStack ( WordStack , Int ) [function, total, klabel(sizeWordStack), smtlib(sizeWordStack)]
  // -----------------------------------------------------------------------------------------------------------------------
```

Comment:
This seems a strange modification.... Why would we want to give this the same label as the single-argument variant?